src/query/interpreter.cpp

```diff
@@ -695,7 +708,8 @@ Callback HandleStreamQuery(StreamQuery *stream_query, const Parameters &paramete
       return callback;
     }
     case StreamQuery::Action::SHOW_STREAMS: {
-      callback.header = {"name", "type", "batch_interval", "batch_size", "transformation_name", "owner", "is running"};
+      callback.header = {"name", "type", "batch_size", "batch_interval", "transformation_name", "owner", "is running"};
```

Comment:
The `batch_interval` is pushed to the result first, so I don't understand this change.

Reply:
No idea what was behind this change, I'll revert.